### PR TITLE
OSDOCS#6367:Installing specific Operator version in web console

### PR DESCRIPTION
**Pull request by opayne1**
OSDOCS#6367:Installing specific Operator version in web console

Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-6367

Link to docs preview:
https://64953--docspreview.netlify.app/openshift-enterprise/latest/operators/admin/olm-adding-operators-to-cluster#olm-installing-specific-version-web-console_olm-adding-operators-to-a-cluster

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by sg00dwin — CHANGES_REQUESTED** [review]

**Comment by XiyunZhao** [comment]
Hi @opayne1, the content is looks good for me
/qe-approved

File: modules/olm-installing-specific-version-web-console.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+. From the selected Operator, you can select a *Channel* and *Version* from the lists.
++
+[NOTE]
+====
+The version selection will default to the latest version for the channel selected. If the latest version for the channel is selected, the _Automatic approval_ strategy is enabled by default. Otherwise _Manual approval_ is required when not installing the latest version for the selected channel.
+
+_Manual approval applies to all operators installed in a namespace_
+
+Installing an Operator with manual approval causes all Operators installed within the namespace to function with the _Manual approval_ strategy and will be updated altogether. Install Operators into separate namespaces for updating independently.
```
Comment (mburke5678) [inline modules/olm-installing-specific-version-web-console.adoc:29]:
```suggestion
Installing an Operator with manual approval causes all Operators installed within the namespace to function with the _Manual approval_ strategy and all Operators are be updated together. Install Operators into separate namespaces for updating independently.
```

File: modules/olm-installing-specific-version-web-console.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+====
+
+. Click *Install*
+
+.Verification
+
+* When the operator is installed, the metadata will indicate which channel and version are installed.
```
Comment (mburke5678) [inline modules/olm-installing-specific-version-web-console.adoc:36]:
```suggestion
* When the operator is installed, the metadata indicates which channel and version are installed.
```

**Comment by mburke5678** [comment]
@opayne1 A few nits, and an ignorant question. Otherwise LGTM

File: modules/olm-installing-specific-version-web-console.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+. Select an Operator you want to install.
+
+. From the selected Operator, you can select a *Channel* and *Version* from the lists.
++
+[NOTE]
+====
+The version selection will default to the latest version for the channel selected. If the latest version for the channel is selected, the _Automatic approval_ strategy is enabled by default. Otherwise _Manual approval_ is required when not installing the latest version for the selected channel.
+
+_Manual approval applies to all operators installed in a namespace_
```
[the remark this answers is not in the document]
Comment (opayne1) [reply modules/olm-installing-specific-version-web-console.adoc:27]: You are correct. I was getting the lowercase italics confused with another UI element that requires that. Thanks!

**Pull request merged**